File: Aula 04 - Estruturas de Decisao/cpp_logic/switch.cpp
#include <iostream>
using namespace std;

int main() {
    int codigo_status;
    cout << "Digite o código de status (1= Disponínel, 2= Reposição, 3= Descontinuado): ";
    cin >> codigo_status;

    switch (codigo_status) {
        case 1: cout << "Status: Disponível." << endl;
            break;
        case 2: cout << "Status: Em Reposição." << endl;
            break;
        case 3: cout << "Status: Desconinuado." << endl;
            break;
        default: cout << "Código de status inválido." << endl;
    }


    return 0;
}

File: Aula 04 - Estruturas de Decisao/java_logic/switch.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite o código de status (1=Disponínel, 2=Reposição, 3=Descontinuado): ");
        int codigo_status = scanner.nextInt();

        switch (codigo_status) {
            case 1:
                System.out.println("Status: Disponível.");
                break;
            case 2:
                System.out.println("Status: Em Reposição.");
                break;
            case 3:
                System.out.println("Status: Descontinuado.");
                break;
            default:
                System.out.println("Código de status inválido.");
        }
        
        scanner.close();
    }
    
}